.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@ojkelly/plugin-build.cjs
+    spec: "https://yarn.build/latest"
```

Comment:
can you explain why do we need this plugin?

Reply:
Yes, RDIL and I had been talking with Owen on Discord and he recommended tophat/monodeploy instead

Reply:
👋 yeah for sending the packages to npm, tophat/monodeploy is probably a good choice.

yarn.build is more like a bazel/buck, so can build local workspaces in the correct topological order with as much concurrency as possible, and it keeps a record of what's been built so it only rebuild what's changed.

It's also great for doing the same with tests `yarn test` and you'll find it most powerful when each workspace is treated as a self contained unit.

If you pair it with yarn's vendoring, then you get the speed and soundness from the big tools like bazel/buck.